lib/main_common.pm

```diff
@@ -86,6 +88,13 @@ sub setup_env {
     }
 }
 
+sub any_desktop_is_applicable() {
+    return get_var("DESKTOP") !~ /textmode/;
+}
+
+sub console_is_applicable() {
```

Comment:
yes, propose renaming but outside this PR. What you do is extract an existing method into a common package that is included from multiple main.pm's. That makes sense and keep it for now.